sentry/src/main/java/io/sentry/TracesSamplingDecision.java

```diff
@@ -9,14 +9,27 @@ public final class TracesSamplingDecision {
 
   private final @NotNull Boolean sampled;
   private final @Nullable Double sampleRate;
+  private final @NotNull Boolean profileSampled;
+  private final @Nullable Double profileSampleRate;
+
+  public TracesSamplingDecision(@NotNull Boolean sampled) {
```

Comment:
```suggestion
  public TracesSamplingDecision(final @NotNull Boolean sampled) {
```

Please add all the missing `final` to this PR, and missing `@Nullable` and @NotNull`.